content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+  job position is selected, after the card is created, the card will appear in the selected job
+  position.
+
+After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
+new blank applicant card appears. Click either the :guilabel:`🗑️ (delete)` icon or click anywhere
+on the screen to close the card.
+
+.. image:: recruitment/quick-add.png
+   :align: center
+   :alt: All the fields for a new applicant form entered when using the Quick Add option.
+
+Create
+******
+
+Add a new applicant including all the relevant information using the :guilabel:`Create` button. If
+not already in the kanban view for the job position to add an applicant to, navigate to the main
+recruitment dashboard, and click the :guilabel:`New Applicants` smart button on the job position
+card that the applicant should be added to. Then, click the :guilabel:`Create` smart button in the
+top left of the kanban view and an applicant form loads.
+
+Certain fields on the applicant card may be pre-populated, depending on how the job position is
+configured. Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter` field, are
+all pre-populated, but this may vary.
+
+Enter the following information on the new applicant form. Note that not all fields listed below may
+be visible. Depending on installed applications and configurations, some fields may not be
+displayed.
+
+Applicant section
+^^^^^^^^^^^^^^^^^
+
+- :guilabel:`Subject/Application Name`: This is the only required field. Enter the title for the
+  card. Typically this is the applicant's name and job position being applied to, for example:
+  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is
+  visible on the individual card.
+- :guilabel:`Applicant's Name`: Enter the applicant's name.
+- :guilabel:`Email`: Enter the applicant's email address.
+- :guilabel:`Phone`: Enter the applicant's phone number.
+- :guilabel:`Mobile`: Enter the applicant's mobile number.
+- :guilabel:`Degree`: Select the applicant's highest level of education earned from the drop-down
+  menu. Options are :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
+  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates graduating the highest
+  level of school before a Bachelor's degree, such as a high school or secondary school diploma,
+  depending on the country.
+- :guilabel:`Tags`: Select as many tags as desired from the drop-down menu. To add a tag that does
+  not exist, type in the tag name, the click :guilabel:`Create "new tag"`.
+- :guilabel:`Interviewer`: Select the employee who will be conducting the interview(s) from the
+  drop-down menu. Anyone in the company can be selected as the interviewer, they do not have to be
+  a recruiter.
+- :guilabel:`Recruiter`: Select the person responsible for the entire recruitment process for the
```

Comment:
I dug in more- and I updated this. Based on permissions- which I assumed but was told otherwise- and did a second test, with permissions- to confirm how it works.